curriculum/challenges/english/07-scientific-computing-with-python/learn-classes-and-objects-by-building-a-sudoku-solver/660a737f0f72b51de361051c.md

```diff
@@ -0,0 +1,83 @@
+---
+id: 660a737f0f72b51de361051c
+title: Step 54
+challengeType: 20
+dashedName: step-54
+---
+
+# --description--
+
+The `:=` operator gives you the ability of assigning variables in the middle of an expression. The syntax is: `name := val`, where `name` is the variable name and `val` is the variable value.
+
+This construct is formally named *assignment expressions*, while the `:=` operator is commonly referred to as the *walrus* operator.
+
+Since you are going to need the `self.find_empty_cell()` call more than once, assign it to a variable `next_empty` by usign the walrus operator. Then, enclose the assignment between a pair of parentheses.
```

Comment:
```suggestion
Since you are going to need the `self.find_empty_cell()` call more than once, assign it to a variable `next_empty` by using the walrus operator. Then, enclose the assignment between a pair of parentheses.
```